services/app/src/components/chessBoard/Square.jsx

```diff
@@ -32,22 +33,16 @@ const Square = ({
     ? 3
     : 2;
 
-  const updatePopover = bool => {
-    if (items.length && bool || !bool) {
-      setShowPopover(bool);
-    }
-  }
-
   return (
     <div
       data-testid={`chess-tile-${row}-${column}`}
       className={classNames("square", {
         'is-empty-tile': !items.length && row !== board.length - 1 && column !== 0
       })}
-      onMouseEnter={() => updatePopover(true)}
-      onMouseLeave={() => updatePopover(false)}
+      onMouseEnter={() => {setIsHovered(true)}}
+      onMouseLeave={() => {setIsHovered(false)}}
     >
-      {!!items.length && (
+      {(isHovered && (!items.length && row !== board.length - 1 && column !== 0) && (gameState === GameState.GAME_IN_PROGRESS)) && (
```

Comment:
Looking at the ACs in the ticket column letter and row number should be hidden only when the game is finished. So lets change ```gameState === GameState.GAME_IN_PROGRESS```  to ```gameState !== GameState.GAME_FINISHED```.